src/components/TradeWidget/index.tsx

```diff
@@ -607,7 +591,7 @@ const TradeWidget: React.FC<TradeWidgetProps> = ({
   const onSelectChangeReceiveToken = onSelectChangeFactory(setReceiveToken, sellTokenBalance)
 
   const tokenAddressesToAdd: string[] = useMemo(
-    () => preprocessTokenAddressesToAdd([sellTokenSymbol, receiveTokenSymbol], networkIdOrDefault),
+    () => preprocessTokenAddressesToAdd([sellToken?.symbol, receiveToken.symbol], networkIdOrDefault),
```

Comment:
why is `sellToken?.symbol` but not receiveToken?

Reply:
good question, I would swear the two of them were nullables, let me check what i did wrong